data/faq.md

```diff
@@ -31,40 +53,40 @@ A. To get an idea of what to expect, please have a look at the
 
 ## **Q. When do you accept talk proposals?**
 
-A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) opened on 8 March.
+A. The [Call for Proposals](https://program.europython.eu/europython-2022/cfp) (CFP) ran between 8 March and 3 April.
+
+We are thrilled to have received a record high of 429 proposals on a diverse range of topics.
+
+## **Q. How do you review and select talks?**
+
+A. We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus after Call for Proposals (CFP) finishes, submissions go through two round of unbiased reviews: speakers' names will not be displayed.
+
+- community voting: open to all 2015-2022 EuroPython conference ticket holders + 2022 talk proposal submitters. Submissions are in random order for each voter. Please check out our [talk voting page](https://ep2022.europython.eu/voting) for details.
+
+- panel feedback: we invite a diverse group of experienced reviewers to refine & improve on submissions within their areas of expertise. The feedback helps the programme team make informed decisions while evaluating the quality of each submission. [Submit your interest here](https://t.co/JyOdGb71f8)
 
-It has been extended for another week. New deadline: **3 April AoE**.
+The programme team will curate the final programme based on input from community voting, panel feedback and other editorial criteria such as the diversity of topics and speakers.
 
-We are thrilled
-to receive proposals on any topic that could be of interest to our community.
+[Subscribe to receive email updates](https://blog.europython.eu/#/portal/signup)
 
-Especially if you are considering becoming a speaker for the first time, or if
-you are from an underrepresented group, we strongly encourage you to send in
-your proposals. We ran a
-[Speaker Mentorship Programme](https://ep2022.europython.eu/mentorship) and matched 15 mentees and mentors. Check out the [AMA session](https://www.youtube.com/watch?v=18donGcl0Jg) as part of the Mentorship Programme. Email us at ([program@europython.eu](mailto:program@europython.eu)) to tell us what other broader support you might need!
+## **Q. What support do you give to speakers?**
+A. If accepted, speakers are eligible for a free conference ticket and trainers a free combined ticket. In addition, speakers are encouraged to apply for additional [Financial Aid](https://ep2022.europython.eu/finaid), should there be a need.
 
-If you have something in mind, there is no better time to start preparing your
-proposal! For more insights, please have a look at the
-[EP2019 CFP page](https://ep2019.europython.eu/events/call-for-proposals/),
-which was our last in-person conference. The overall structure for EP2022 remains the same.
+We want to give extra support to first-time speakers, especially the ones from an underrepresented group. It is in this spirit that we ran a
```

Comment:
We give support to first-time speakers and actively encourage, accommodate and welcome the participation of those from under-represented groups in tech. For example, our Speaker Mentorship Programme matched 15 mentees with mentors. Check out... etc

Reply:
This is so much better. Thank you!